SwanCustomEnvironments/swancustomenvironments/scripts/builders/accpy.sh

```diff
@@ -11,4 +14,4 @@ eval "${ACTIVATE_ENV_CMD}"
 
 # Install packages in the environment and the same ipykernel that the Jupyter server uses
 _log "Installing packages from ${REQ_PATH}..."
-pip install -r "${REQ_PATH}" "ipykernel==${IPYKERNEL_VERSION}" | tee -a ${LOG_FILE}
+uv pip install -i https://acc-py-repo.cern.ch/repository/vr-py-releases/simple --allow-insecure-host acc-py-repo.cern.ch -r "${REQ_PATH}" ${SPARKCONNECTOR} ${SPARKMONITOR} ${PY4J} | tee -a ${LOG_FILE}
```

Comment:
This is not nice, is it? Is there no way to make uv use the repositories?

Also, what are SPARKCONNECTOR and MONITOR doing here?

Reply:
Theres a `uv` option to specify a config file. I provided the one that accpy generates for the created environment but it complained about its syntax. So I just extracted those two.

The Spark extensions was a mistake doing the commits, cause that line was changed for two different purposes. Those extensions are only installed when they are defined, this is, when NXCALS flag is enabled

Reply:
Would it make sense to ping Philip about that syntax error when trying to pass the file to uv? Perhaps the file can be adapted?

Reply:
```bash
(test) [acc-py] (base) [jovyan@78db9ba4f66e ~]$ uv pip install --help | grep config
  -C, --config-setting <CONFIG_SETTING>
      --config-file <CONFIG_FILE>  The path to a `uv.toml` file to use for configuration [env: UV_CONFIG_FILE=]
      --no-config                  Avoid discovering configuration files (`pyproject.toml`, `uv.toml`) [env: UV_NO_CONFIG=]

(test) [acc-py] (base) [jovyan@78db9ba4f66e ~]$ cat test/pip.conf 
#
# pip config file for the CERN Acc-Py distribution
#
[global]
timeout = 20
index-url = https://acc-py-repo.cern.ch/repository/vr-py-releases/simple
trusted-host=acc-py-repo.cern.ch
disable-pip-version-check=1
no-python-version-warning=1
root-user-action=ignore
user=no

(test) [acc-py] (base) [jovyan@78db9ba4f66e ~]$ uv pip install --config-file test/pip.conf requests
error: Failed to parse: `test/pip.conf`
  Caused by: TOML parse error at line 6, column 13
  |
6 | index-url = https://acc-py-repo.cern.ch/repository/vr-py-releases/simple
  |             ^
invalid string
expected `"`, `'`

(test) [acc-py] (base) [jovyan@78db9ba4f66e ~]$ 
```

it will complain about all the elements not having quotes. inserting them manually, it complains about that `[global]` section tag. so i just gather the two we need